homeassistant/setup.py

```diff
@@ -357,11 +357,10 @@ async def async_process_deps_reqs(
     if failed_deps := await _async_process_dependencies(hass, config, integration):
         raise DependencyError(failed_deps)
 
-    if not hass.config.skip_pip and integration.requirements:
```

Comment:
`async_get_integration_with_requirements` checks if `skip_pip` is set, we don't need to check it here too

Reply:
`async_process_deps_reqs` is also used in other places that then needs to honor skip_pip